File: typescript/packages/management/src/tools/list_sandbox_connections.ts
import type { ToolContext, ToolHandler } from '../registrar.js';

export const listSandboxConnections: ToolHandler = async (
  ctx: ToolContext,
  input: Record<string, unknown>,
) => {
  return ctx.client.sandboxConnections.list(ctx.sandboxId, {
    include_revoked: input.include_revoked === true,
  });
};

File: python/copass-management/src/copass_management/tools/list_sandbox_connections.py
from __future__ import annotations

from typing import Any, Dict, TYPE_CHECKING

if TYPE_CHECKING:
    from copass_management.registrar import ToolContext


async def list_sandbox_connections(ctx: "ToolContext", input: Dict[str, Any]) -> Any:
    return await ctx.client.sandbox_connections.list(
        ctx.sandbox_id,
        include_revoked=bool(input.get("include_revoked")),
    )
